base_layer/key_manager/src/lib.rs

```diff
@@ -11,3 +11,25 @@ pub mod mnemonic_wordlists;
 #[allow(clippy::unused_unit)]
 #[cfg(feature = "wasm")]
 pub mod wasm;
+
+use tari_common::hashing_domain::*;
+
+/// The key manager domain separated hashing domain
+/// Usage:
+///   let hash = comms_dht_hash_domain().digest::<Blake256>(b"my secret");
+///   etc.
+pub fn base_layer_key_manager_mac_generation() -> HashingDomain {
+    HashingDomain::new("base_layer.key_manager.cipher_seed.mac_generation")
+}
+
+pub fn base_layer_key_manager_argon2_encoding() -> HashingDomain {
+    HashingDomain::new("base_layer.key_manager.cipher_seed.argon2_encoding")
+}
+
+pub fn base_layer_key_manager_chacha20_encoding() -> HashingDomain {
+    HashingDomain::new("base_layer.key_manager.cipher_seed.chacha20_encoding")
+}
```

Comment:
I see these functions are not being used in this PR; I suppose they were created for use elsewhere?

Reply:
Ah I see, used in #4296.